pkgs/development/python-modules/hdate/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, astral
+, buildPythonPackage
+, fetchFromGitHub
+, poetry-core
+, pytestCheckHook
+, pythonOlder
+, pytz
+}:
+
+buildPythonPackage rec {
+  pname = "hdate";
+  version = "0.10.2";
+  disabled = pythonOlder "3.6";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "py-libhdate";
+    repo = "py-libhdate";
+    rev = "v${version}";
+    sha256 = "07b0c7q8w6flj4q72v58d3wymsxfp5qz8z97qhhc2977mjx5fsxd";
+  };
+
+  nativeBuildInputs = [
+    poetry-core
+  ];
+
+  propagatedBuildInputs = [
+    astral
+    pytz
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+  ];
+
+  postPatch = ''
+    substituteInPlace pyproject.toml --replace "^2020.5" "^2021"
```

Comment:
This substitution lacks a bit of context.

```
pytz = "^2020.5"
```

Reply:
The intention was good but doesn't work well with the versioning schema of `pytz` as far as I can tell. We need something like >=2020.5 that is not limited to <2021.

Reply:
So `>=2020.5`?

Reply:
That will work :smile: